File: GTTS/INL.ReferenceService/Models/Units_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.ReferenceService.Models
{
    public class Units_Item
	{
		public Int64 UnitID { get; set; }
		public Int64? UnitParentID { get; set; }
		public int CountryID { get; set; }
		public Int64 UnitLocationID { get; set; }
		public int ConsularDistrictID { get; set; }
		public string UnitName { get; set; }
		public string UnitNameEnglish { get; set; }
		public bool IsMainAgency { get; set; }
		public Int64 UnitMainAgencyID { get; set; }
		public string UnitAcronym { get; set; }
		public string UnitGenID { get; set; }
		public int UnitTypeID { get; set; }
		public int? GovtLevelID { get; set; }
		public int? UnitLevelID { get; set; }
		public int VettingTypeID { get; set; }
		public int VettingActivityTypeID { get; set; }
		public int ReportingTypeID { get; set; }
		public Int64 UnitHeadPersonID { get; set; }
		public int? UnitHeadJobTitleID { get; set; }
		public int? UnitHeadRankID { get; set; }
		public Int64 HQLocationID { get; set; }
		public string POCName { get; set; }
		public string POCEmailAddress { get; set; }
		public string POCTelephone { get; set; }
		public string VettingPrefix { get; set; }
		public bool HasDutyToInform { get; set; }
		public bool IsLocked { get; set; }
		public int ModifiedByAppUserID { get; set; }
		public DateTime ModifiedDate { get; set; }

		public string Breakdown { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.ReferenceService.Models/units_item.ts



export class Units_Item  {
  
	public UnitID: number = 0;
	public UnitParentID?: number;
	public CountryID: number = 0;
	public UnitLocationID: number = 0;
	public ConsularDistrictID: number = 0;
	public UnitName: string = "";
	public UnitNameEnglish: string = "";
	public IsMainAgency: boolean = false;
	public UnitMainAgencyID: number = 0;
	public UnitAcronym: string = "";
	public UnitGenID: string = "";
	public UnitTypeID: number = 0;
	public GovtLevelID?: number;
	public UnitLevelID?: number;
	public VettingTypeID: number = 0;
	public VettingActivityTypeID: number = 0;
	public ReportingTypeID: number = 0;
	public UnitHeadPersonID: number = 0;
	public UnitHeadJobTitleID?: number;
	public UnitHeadRankID?: number;
	public HQLocationID: number = 0;
	public POCName: string = "";
	public POCEmailAddress: string = "";
	public POCTelephone: string = "";
	public VettingPrefix: string = "";
	public HasDutyToInform: boolean = false;
	public IsLocked: boolean = false;
	public ModifiedByAppUserID: number = 0;
	public ModifiedDate: Date = new Date(0);
	public Breakdown: string = "";
  
}


